ksql-engine/src/test/java/io/confluent/ksql/util/FakeKafkaTopicClient.java

```diff
@@ -52,6 +56,16 @@ public int getNumPartitions() {
     public short getReplicatonFactor() {
       return replicatonFactor;
     }
+
+    public TopicDescription getDescription() {
+      List<TopicPartitionInfo> partitionInfoList = new LinkedList<>();
```

Comment:
Why `LinkedList`?

Reply:
No special reason. Is there a more idiomatic way to construct a implementation-insensitive mutable list?